### ISSUE TITLE
Fastboot: Unknown error:  downloading 'boot.img'...OKAY [  0.360s]booting...FAILED (remote: bootimage: incomplete or not signed)finished. total time: 0.361s

### ISSUE THREAD

**Issue by bosumtwi** [ubports/ubports-installer#363]
*Automatically generated

Hi,
I am trying to install ubunto touch for the first time.
I am using a fairphone 2 and I am tryng to install it from a Windows 10 machine.
I have tried different installers (version 12, 13 and 20).
Now, with the version ubports-installer.0.1.20-beta.exe I get the fastboot error message.
As a new user, I would need some help to get the ubunto touch running on my new fairphone.
Hope somebody can help.
Thanks a lot in advance
Sascha

**Comment by bosumtwi** [comment]
**Update:**
Ok. Here my solution, how I managed to install Ubuntu touch on my Fairphone:
I made an Ubuntu live DVD, started Ubuntu, installed the ubport-installer and got it installed.
In between, the ubport-installer was hanging, after downloading the files. I installed adb-android-tools and in the second try it was working. But I am uncertain if this was actually the reason.
At least. Ubunto touch is running fine now :-)
